dataset-snippets/codegen_sample.r

```diff
@@ -66,21 +66,39 @@ AND (
                 )
  ", sep="")
 
-# Extract the cohort to Cloud Storage as CSV.
-out_dir_15947426 <- "out_15947426"
-export_15947426_glob <- paste(out_dir_15947426, "/export15947426-*.csv")
-export_15947426_path <- paste(Sys.getenv("WORKSPACE_BUCKET"), "/bq_exports/", export_15947426_glob, sep="")
+# Formuate a Cloud Storage destination path for the data exported from BigQuery.
+# NOTE: By default data exported multiple times on the same day will overwrite older copies.
+#       But data exported on a different days will write to a new location so that historical
+#       copies can be kept as the dataset definition is changed.
+survey_export_87319558_path <- file.path(
+    Sys.getenv("WORKSPACE_BUCKET"),
+    "bq_exports",
+    Sys.getenv("OWNER_EMAIL"),
+    strftime(lubridate::now(), "%Y%m%d"),  # Comment out this line if you want the export to always overwrite.
+    "survey_87319558",
+    "survey_87319558_*.csv")
```

Comment:
nit: feels slightly redundant with the dir name - maybe just `*.csv`?

Reply:
I think it's important to keep the context in the filename in case folks copy them to the VM disk (perhaps without recreating the directory structure). I also think it's important to have the survey export in a different GCS folder than the measurements/person, etc.. in case the number of files is large. Hence the current redundancy in the path.

A different folding naming suggestion might improve it, but I would recommend against `*.csv` as the filename we ask BQ export to use.

Reply:
OK, sounds reasonable.